lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -425,10 +425,11 @@ LogicalResult static applyNoBlackBoxStyleDataTaps(const AnnoPathValue &target,
       if (!moduleTarget)
         return failure();
       AnnoPathValue internalPathSrc;
-      sendVal = lowerInternalPathAnno(
-          internalPathSrc, *moduleTarget, target, internalPathAttr,
-          wireTarget->ref.getOp()->getResult(0).getType().cast<FIRRTLType>(),
-          state);
+      auto targetType = wireTarget->ref.getType().cast<FIRRTLBaseType>();
+      if (wireTarget->fieldIdx)
```

Comment:
Not a nit, but suggestion: WDYT about always feeding this through `getFinalTypeByFieldID` instead of doing the check first?
(this is how we use `getValueByFieldID`, for example)

This code already has a _ton_ of conditional paths.

Erm, nevermind I see this was merged.  All good! LGTM, thanks for quick fix!